### PR TITLE
COMMERCE-6036

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
Tested and it works just fine. Is it doable, though?

@ivicac @alecsloan

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/src/main/resources/com/liferay/commerce/theme/speedwell/site/initializer/internal/dependencies/journal_articles/SPEEDWELL_ACCOUNT.ftl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="account-dashboard">
 	<h2 class="account-dashboard__title">${PageTitle.getData()}</h2>
 
-	<a class="account-dashboard__pill" href="${NavigationItemLinkOrders.getFriendlyUrl()}">
+	<a class="account-dashboard__pill" href="${themeDisplay.getPathFriendlyURLPublic() + themeDisplay.getScopeGroup().getFriendlyURL() + "/my-orders"}">
```
Comment (ivicac) [inline modules/apps/commerce/commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/src/main/resources/com/liferay/commerce/theme/speedwell/site/initializer/internal/dependencies/journal_articles/SPEEDWELL_ACCOUNT.ftl:4]: @gianmarcobrunialti assign "themeDisplay.getPathFriendlyURLPublic() + themeDisplay.getScopeGroup().getFriendlyURL()" to a varibale and then use it for all links

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:forward

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**